src/App/Command/DisableCommand.php

```diff
@@ -65,10 +70,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         fwrite($handle, '];' . "\n");
         fclose($handle);
 
-        if (!empty($disabledPackages)) {
-            $io->success("Disabled packages:\n — " . implode("\n — ", $disabledPackages));
-        } else {
+        if (empty($alreadyDisabledPackages) && empty($disabledPackages)) {
             $io->info('Packages not found.');
```

Comment:
Similar to https://github.com/yiisoft/yii-dev-tool/pull/219/files#r947529868.

Reply:
Done.